web/app/js/components/Sidebar.jsx

```diff
@@ -180,6 +180,25 @@ class Sidebar extends React.Component {
                 : null
             }
 
+            <Menu.Item className="sidebar-menu-item" key="/authorities">
```

Comment:
was this intentional? i get a 404 on `/authorities`

Reply:
Yeah, it will 404 until I merge https://github.com/runconduit/conduit/pull/1189